### ISSUE TITLE
Tc resistance lock

### ISSUE THREAD

**Issue by Hansson0728** [arafey/openevic#18]
Hi iam using a rx2/3 with a dual coil setup,
I have turned auto check to no, because i usally set my own resistance,  but i keeps reverting back to the auto read value even though i lock the resitance and have the check turned off. 
Is this something that is known since before ?

**Comment by arafey** [comment]
Is it the same as [this](https://github.com/ClockSelect/myevic/issues/313)?

**Comment by Hansson0728** [comment]
No, i can change it, so it says no, but it seems the actual function is still turned on.

**Comment by arafey** [comment]
Alright, I'll need to take a look. It's possible that this function is either not fully implemented or faulty. I haven't built or used a coil yet that triggers the auto-check to kick me into power mode so I will need to do so in order to test/build/fix this function.

**Comment by Hansson0728** [comment]
I dont het kicked out of tc though, it just the resitane hanges even if i set it manually, if my oil goes in at 0,480 and i want to set it to 0,500 then after reboot it goes back to .480 even if i locked it

**Comment by arafey** [comment]
My most recent response on #22 applies here as well.

**Comment by Hansson0728** [comment]
Did have similar problems on myvic to actually, ive worked ariund it by just upping my temp setting instead of upping coil resistance

**Comment by arafey** [comment]
Disabling auto check is for forcing TC mode even if the mod thinks that you're using a non-TC coil. So if you're not getting kicked out of TC then it doesn't have to do with auto check (unless you want it to kick you out, you're using Kanthal, and yet it's sticking to TC with auto check enabled, which would be an issue also).

By "reboot" do you mean removing the battery and reinserting it or do you mean clicking it 5 times to off and then 5 times to on?

If I understand you correctly, this issue has existed in exactly the same form since myevic? Or has it changed / gotten worse?

**Comment by Hansson0728** [comment]
Its basically the same, sometimes it works, sometimes it resets the resitance,  well since i dont get kicked out that functions is probably working as it should, might be something else, by reboot i mean ive tried both,

**Comment by arafey** [comment]
This is a guess as I haven't looked at the responsible code, but it's possible that the device detects a change in resistance and reverts your offset. 

In the example you gave, you said that the coil goes in at 0.480Ω, you offset it to 0.500Ω, and then you get 0.480Ω again after a restart or reset. 

Is that *exactly* right? Or is it more like you start with 0.480Ω, do the offset, but then it ends up at 0.478Ω or 0.482Ω after reboot (ie. some number that is not the exact same resistance as what you started with)?

**Comment by Hansson0728** [comment]
It allways reverts back to the same number as far as ive noticed

**Comment by arafey** [comment]
Does the resistance on the screen show 0.000Ω before changing to 0.480Ω? Watch the screen as it turns on to make sure because it will only show 0 for less than a second. I'm trying to determine if the problem is in the logic for detecting a new coil being attached or in saving the offset to memory properly.

**Comment by Hansson0728** [comment]
As far as i can see right now it does not go down to zero, i will watch this closely today

**Comment by arafey** [comment]
Thank you. I'm still familiarizing myself with the code so it's hard to tell the difference between a hidden feature and an actual bug. I'll wait for your results before continuing.

**Comment by arafey** [comment]
Have you checked this yet?

**Comment by Hansson0728** [comment]
Yes i have monitored it, and from what i can tell it does not go to zero on startup, and it allways reverts to exactly the same value (different value depending on tank)
